CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.20.0](https://github.com/redhat-developer/quarkus-ls/milestone/23?closed=1) (October 17, 2024)

```

Comment:
I would get rid of the 'feat' / 'fix' prefixes. We know what they are based on the headings.

Reply:
fixed